crossplane/aws-providers/aws-providerconfig.yml

```
@@ -0,0 +1,11 @@
+apiVersion: aws.upbound.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: aws-secret
+      key: creds
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)